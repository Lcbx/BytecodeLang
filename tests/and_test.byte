4<6 and 45==45 and 23==32
'should execute till end of line 1'
23==32 and 45==45 and 4<6
'should jump to line 5 after 23==32'
23==23 and 45!=45 and 4<6
'should jump to line 7 after 45!=45'
34 == 11 * 3 and 23 == 23
'should jump to value 1024 after 34 == 11 * 3'
1024
