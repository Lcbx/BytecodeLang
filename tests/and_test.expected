compile_Out:
Error line 7, column 18 : illegal token : '"'
Error line 7, column 18 : and expression missing right operand
Error line 7, column 18 : illegal token : '"'
3 errors
__SECTION__
compile_Err:

__SECTION__
simulation_Out:

__SECTION__
simulation_Err:
Traceback (most recent call last):
  File "C:\Users\lucco\Desktop\tests\Lang\compiler\vm_simulator.py", line 22, in <module>
    opName = opcodes[opcode].name
KeyError: 49